RELEASING.md

```diff
@@ -2,45 +2,18 @@
 
 This page describes how to cut releases of our plugin. It's strictly for the owners of the project.
 
-1. Create a new branch as if you were starting a normal bugfix PR
+1. Create a new branch named release_v{$version}.  It's critical that you create the branch according to this schema or the release will fail.
 
```

Comment:
The release will fail if the local branch isn't up to date, but fair point
that the instructions should help the user make sure they are up to date
before they start.

On Thu, Feb 4, 2016 at 1:40 PM, Appu Goundan <notifications@github.com>
wrote:

> In RELEASING.md
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/411#discussion_r51917033>
> :
>
> > @@ -2,45 +2,18 @@
> >
> >  This page describes how to cut releases of our plugin. It's strictly for the owners of the project.
> >
> > -1. Create a new branch as if you were starting a normal bugfix PR
> > +1. Create a new branch named release_v{$version}.  It's critical that you create the branch according to this schema or the release will fail.
> >
>
> Is there concern local branches wont be up to date?
>
> how about something like?
>
> git fetch
> git checkout -b release_v1.0 origin/master
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/411/files#r51917033>
> .
>